src/WebJobs.Extensions.DurableTask/ETWEventListener.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    internal class ETWEventListener : EventListener
+    {
+        private ILogger logger;
+
+        public ETWEventListener(ILogger logger)
+        {
+            this.logger = logger;
+        }
+
+        protected override void OnEventWritten(EventWrittenEventArgs eventData)
+        {
+            if (DurableTaskExtension.InLinux)
+            {
+                // TODO: Not sure that LogLevel here is right
+                this.logger.Log(LogLevel.Information, eventData.EventId, eventData, null, null);
```

Comment:
Yup! You're right! I haven't updated my ILogger implementation to take care of that yet. That's the place where I'm thinking it would make most sense to include that logic